step-api/step-api-keyword/src/main/java/step/handlers/javahandler/Keyword.java

```diff
@@ -49,4 +54,8 @@
 	 */
 	String schema() default "";
 
+	/**
+	 * @return the call timeout for this keyword
```

Comment:
@jeromebrongniart The unit should be specified in the comment

Reply:
Done: https://github.com/exense/step-api/pull/18/commits/5665e48f4e4eb70b85d067c196c1dd4ff3d6463a